Ago-Dic-2020/sena-martinez-angel-david/Practica2.py

```diff
@@ -0,0 +1,37 @@
+class Persona():
+    def __init__(self, edad, peso, altura):
+        self.edad = edad
+        self.peso = peso
+        self.altura = altura
+
+    def _str_(self):
+        return f"Edad: {self.edad} años\nPeso:{self.peso}\nAltura:{self.altura}cm"
+    
+    def pesoideal(self):
```

Comment:
Nombres de funciones y variables deben de ser `snake_case` por recomendaciones del [estándar oficial](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)